common/ta_errors.h

```diff
@@ -156,6 +156,8 @@ typedef enum {
   /**< Can't find message in the assign bundle */
   SC_MAM_INVAID_CHID_OR_EPID = 0x0E | SC_MODULE_MAM | SC_SEVERITY_FATAL,
   /**< Failed to add trusted channel ID or endpoint ID */
+  SC_MAM_STARTING_CHID = 0x0F | SC_MODULE_MAM | SC_SEVERITY_FATAL,
```

Comment:
How about changing to `SC_MAM_EXCEEDED_CHID_ITER`?

Reply:
OK